doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -5428,6 +5428,18 @@ right hand side double , view hint declaration see :ref:`declaring_new_hints_ssr
 
 prenex implicits declaration see :ref:`parametric_polymorphism_ssr`
 
+Settings
+~~~~~~~~
+
+.. flag:: Debug Ssreflect
+
+   Print debug information on reflect to standard output.
+
+.. flag:: Debug SsrMatching
+
+   Print debug information on SSR matching to standard output.  Has no effect
+   unless the environment variable `SSRMATCHINGDEBUG` exists.
```

Comment:
This note about the env variable is just a bug in the code. I'm pretty sure I wanted to be able to enable the option via an env variable, I guess I should have used a disjunction rather than a conjunction. I'm fine removing this (broken) feature.

Also, I would not be so specific about where prints are performed, eg I see these debug prints in a CoqIDE window too.

Reply:
I was mistaken; the environment variable works correctly.  I'll keep it in the code but remove the mention in the doc unless you suggest otherwise.